README.md

```diff
@@ -2,8 +2,10 @@
 ## Assignment
 Read in some data and graph something in R
 
-Hi :) 
+Hello, a lot of my graphs and the data itslef is in prism files that I no longer have acess to so I shal simply recreate an excel graph that I used to have autogenerate from ELISA data. 
+
+this is the graph! 
```

Comment:
I can't seem to read your graphs but I recreated graphs from excel too with old data and they turned out much more detailed.